src/System.Private.ServiceModel/tests/Scenarios/Client/ChannelLayer/DuplexChannelShapeTests.4.1.0.cs

```diff
@@ -17,11 +16,11 @@ public partial class DuplexChannelShapeTests : ConditionalWcfTest
     // The tests in this file use the IDuplexChannel shape.
 #if FULLXUNIT_NOTSUPPORTED
     [Fact]
-#else
-    [ConditionalFact(nameof(Root_Certificate_Installed))]
 #endif
+    [WcfFact]
+    [Condition(nameof(Root_Certificate_Installed))]
     [OuterLoop]
-    [ActiveIssue(1157)]
```

Comment:
>[ActiveIssue(1157)] [](start = 4, length = 19)

I am in the process of adding back all the deleted unconditional ActiveIssues into the ToF conditional blocks, otherwise ToF tries to run them.  So assume that will be done across the board before the merge.